src/transformStateWithClones.js

```diff
@@ -64,8 +64,31 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const array = [];
```

Comment:
I think we can use better names for variables, array doesn't explain anything.

Reply:
changed